js/app.js

```diff
@@ -458,325 +568,390 @@ $(document).ready(function () {
-      let entryObject = serializeEntries($(this), 'labels');
-      apiCallDeleteEntries(entryObject, 'labels');
+      let entryObject = serializeEntries($(this), "labels");
+      apiCallDeleteEntries(entryObject, "labels");
     });
 
     $('.milestone-entry[action="delete"]').each(function () {
-      let entryObject = serializeEntries($(this), 'milestones');
-      apiCallDeleteEntries(entryObject, 'milestones');
+      let entryObject = serializeEntries($(this), "milestones");
+      apiCallDeleteEntries(entryObject, "milestones");
     });
 
     //To be updated
     $('.label-entry[action="update"]').each(function () {
-      let entryObject = serializeEntries($(this), 'labels');
-      apiCallUpdateEntries(entryObject, 'labels');
+      let entryObject = serializeEntries($(this), "labels");
+      apiCallUpdateEntries(entryObject, "labels");
     });
 
     $('.milestone-entry[action="update"]').each(function () {
-      let entryObject = serializeEntries($(this), 'milestones');
-      apiCallUpdateEntries(entryObject, 'milestones');
+      let entryObject = serializeEntries($(this), "milestones");
+      apiCallUpdateEntries(entryObject, "milestones");
     });
 
     //To be created
     $('.label-entry[action="create"]').each(function () {
-      let entryObject = serializeEntries($(this), 'labels');
-      apiCallCreateEntries(entryObject, 'labels');
+      let entryObject = serializeEntries($(this), "labels");
+      apiCallCreateEntries(entryObject, "labels");
     });
 
     $('.milestone-entry[action="create"]').each(function () {
-      let entryObject = serializeEntries($(this), 'milestones');
-      apiCallCreateEntries(entryObject, 'milestones');
+      let entryObject = serializeEntries($(this), "milestones");
+      apiCallCreateEntries(entryObject, "milestones");
     });
   }
 
   function writeLog(string) {
-    $('#loadingModal .modal-body').append(string + '<br>');
+    $("#loadingModal .modal-body").append(string + "<br>");
   }
 
-  $('#loadingModal').on('hidden.bs.modal', function () {
+  $("#loadingModal").on("hidden.bs.modal", function () {
     isLoadingShown = false;
 
     //reset modal
-    $('#loadingModal .modal-body').text('');
-    $('#loadingModal .modal-body').append('<p>Commiting...');
-    $('#loadingModal .modal-footer').remove();
+    $("#loadingModal .modal-body").text("");
+    $("#loadingModal .modal-body").append("<p>Commiting...");
+    $("#loadingModal .modal-footer").remove();
 
     //reload labels after changes
     clearAllLabels();
     clearAllMilestones();
     let login = getLoginInfo();
-    apiCallGetEntries(login.targetOwner, login.targetRepo, 'labels', 'list');
-    apiCallGetEntries(login.targetOwner, login.targetRepo, 'milestones', 'list');
+    apiCallGetEntries(login.targetOwner, login.targetRepo, "labels", "list");
+    apiCallGetEntries(
+      login.targetOwner,
+      login.targetRepo,
+      "milestones",
+      "list"
+    );
```

Comment:
Is there any reasoning behind the inconsistent formatting?

Reply:
It's all formatted by "Prettier".

It's actually quite consistent. If a line has less than 80 characters, Prettier leaves it alone. But if it exceeds 80 characters, Prettier will try to format them in a reasonable way.

Reply:
I thought that might be the case. Fair play if you want to leave it this way.